python/opensky_api.py

```diff
@@ -77,10 +106,15 @@ def __str__(self):
 class OpenSkyStates(object):
     """ Represents the state of the airspace as seen by OpenSky at a particular time. It has the following fields:
 
-      |  **time** - in seconds since epoch (Unix time stamp). Gives the validity period of all states. All vectors represent the state of a vehicle with the interval :math:`[time - 1, time]`.
-      |  **states** - a list of `StateVector` or is None if there have been no states received
+      |  **time**: `int` - in seconds since epoch (Unix time stamp). Gives the validity period of all states.
+        All vectors represent the state of a vehicle with the interval :math:`[time - 1, time]`.
+      |  **states**: `list` [`StateVector`] - a list of `StateVector` or is None if there have been no states received
     """
     def __init__(self, j):
+        """
+        Initializes the OpenSkyStates object
+        :param dict j: the json represents the state of the airspace as seen by OpenSky at a particular time
```

Comment:
Are you speaking of a JSON string or a Python dict? The character j might be confusing as it is being used for the imaginery unit in Python. I recommend a longer variable name.

Reply:
Dictionary is the right type.